public/scripts/app.js

```diff
@@ -2,6 +2,169 @@ console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
 
-// your code
+  console.log('app.js loaded!');
 
+  $.ajax({
+    method: 'GET',
+    url: '/api/boardgames',
+    success: renderMultipleGames
+  });
+
+// Displays new game form on click
+$('.new-game').on('click', function() {
+  $('.hidden-form').toggle();
+  $('.new-game').toggle();
+})
+
+// Allows user to go back to main view from new game form
+$('#cancel-btn').on('click', function(event) {
```

Comment:
This callback and the above could use the same function (could prevent default on both and call toggle; then you'd only need to write such a function once)